core/src/main/java/org/lflang/generator/CodeBuilder.java

```diff
@@ -126,7 +126,8 @@ public void prSourceLineNumber(EObject eObject, boolean suppress) {
         }
       }
       // Extract the filename from eResource, an astonishingly difficult thing to do.
-      String filePath = CommonPlugin.resolve(eObject.eResource().getURI()).path();
+      String filePath =
+          CommonPlugin.resolve(eObject.eResource().getURI()).path().replace("\\", "\\\\");
```

Comment:
Curious. Why is this necessary?

Reply:
I do not know why this is a problem now but not before. Windows tests were failing that stopped failing after I did this. The path separator needed an extra level of escaping in order to not cause syntax errors in the generated C files.

Reply:
OK :man_shrugging: